src/components/Front/Tile.md

```diff
@@ -0,0 +1,69 @@
+A `<Tile />` is a tilelike front teaser that features text on top of an image, both centered.
```

Comment:
It's not a real issue, but shouldn't that say "features an image on top of text, both centered."?